src/main/java/org/apache/commons/compress/compressors/z/ZCompressorInputStream.java

```diff
@@ -103,12 +103,12 @@ private void reAlignReading() throws IOException {
     @Override
     protected int addEntry(final int previousCode, final byte character) throws IOException {
         final int maxTableSize = 1 << getCodeSize();
-        final int r = addEntry(previousCode, character, maxTableSize);
+        final int idx = addEntry(previousCode, character, maxTableSize);
         if (getTableSize() == maxTableSize && getCodeSize() < maxCodeSize) {
             reAlignReading();
             incrementCodeSize();
         }
-        return r;
+        return idx;
```

Comment:
I am -1 to abbreviated names like "idx". I prefer "index" or, as a fallback for the brevity fans, the classic "i".

Reply:
This suggestion for "idx" comes from a similar expression in UnshrinkingInputStream.java:

    60:        final int idx = addEntry(previousCode, character, MAX_TABLE_SIZE);
    61:        if (idx >= 0) {
    62:            isUsed[idx] = true;
    63:        }
    64:        return idx;
 
The use of "idx" is slightly more common than "index" throughout the project. In this case, there's already a variable that has a similar use, so the system picked it up. Our system doesn't create a new name but it tries to choose the best looking candidate to make the code consistent with a minimal amount of change.